psi4/driver/driver.py

```diff
@@ -988,6 +988,15 @@ def optimize(name, **kwargs):
         custom_gradient = False
 
     return_wfn = kwargs.pop('return_wfn', False)
+    return_history = kwargs.pop('return_history', False)
+
+
+#Be sure to add wavefunction functionality later once the deep copy issues are worked out
```

Comment:
Comments start with a `# ` and are aligned with other code. There also should not be two spaces above this.